docs/user-guide/features/identities/gpg.md

```diff
@@ -61,4 +61,38 @@ user's password and user's secret key.
     ```
     :warning: Depending on your shell version an extra `%` character could appear as shown below, you must disregard this
     character since it's not part of the Initial (one time) AWS Web Console password. 
-    4. If all went well, the decrypted password should be there
\ No newline at end of file
+    4. If all went well, the decrypted password should be there
+
+## Workaround for Mac users
+
+There are some situations where gpg keys generated on Mac don't work properly, generating errors like the following:
+
+```bash
+╷
+│ Error: error encrypting password during IAM User Login Profile (user.lastname) creation: Error encrypting Password: error parsing given PGP key: openpgp: unsupported feature: unsupported oid: 2b060104019755010501
+│ 
+│   with module.user["user.lastname"].aws_iam_user_login_profile.this[0],
+│   on .terraform/modules/user/modules/iam-user/main.tf line 12, in resource "aws_iam_user_login_profile" "this":
+│   12: resource "aws_iam_user_login_profile" "this" {
+│
+```
+
+!!! info ":whale: Docker is required for this workaround."
+    If you don't have docker on your PC, don't worry. You can easily install it following the steps on the [official page](https://docs.docker.com/desktop/mac/install/).
+
+In these cases, execute the following steps:
+
+1. Run an interactive console into an ubuntu container mounting your gpg directory.
+```bash
+docker run --rm -it --mount type=bind,src=/Users/username/.gnupg,dst=/root/.gnupg ubuntu:latest
+```
```

Comment:
Please remember to add steps for`gnupg` installation in the ubuntu system. Namely `apt update` and `apt install gnupg`.
Otherwise the user won't be able to generate the key.
Thanks!